chromadb/test/property/strategies.py

```diff
@@ -39,46 +39,130 @@ def collections(draw) -> Collection:
     """Strategy to generate a set of collections"""
 
     # name = draw(st.from_regex(coll_name_re))
-    name = draw(st.one_of(st.from_regex(_coll_name_re)))
+    name = draw(st.one_of(st.from_regex(_collection_name_re)))
     hypothesis.assume(not _ipv4_address_re.match(name))
     hypothesis.assume(not _two_periods_re.search(name))
 
-    return {"name": name, "metadata": draw(metadata)}
+    return {"name": name, "metadata": draw(collection_metadata)}
+
+
+def one_or_both(strategy_a, strategy_b):
```

Comment:
This is unused?

Reply:
Currently unused - intended for documents and/or embeddings being supplied. Can pull it out, or else leave it in, in anticipation of handling that case when we add generated embedding functions.

Reply:
Going to leave it in for now.